R/FilterState.R

```diff
@@ -20,22 +21,21 @@
 #' - `POSIXct`, `POSIXlt`: `class = DatetimeFilterState`
 #' - all `NA` entries: `class: FilterState`, cannot be filtered
 #' - default: `FilterState`, cannot be filtered
-#' \cr
+#'
 #' Each variable's filter state is an `R6` object which contains `choices`,
 #' `selected`, `varname`, `dataname`, `labels`, `na_count`, `keep_na` and other
 #' variable type specific fields (`keep_inf`, `inf_count`, `timezone`).
-#' Object contains also shiny module (`ui` and `server`) which manages
+#' Object also contains a shiny module (UI and server) which manages the
```

Comment:
```suggestion
#' Object also contains a `shiny` module (UI and server) which manages the
```

@kartikeyakirar can you do GLOBAL FIND for `shiny` and maybe other packages (listed in DESCRIPTION) and make sure they usage of each packages in the documentation is quoted with `\code{}` (or backticks) or `\pkg{}`?

Reply:
Thanks, I have checked all NEST packages, but I will double-check to ensure nothing was missed.

Reply:
yes, just check for packages listed in DESCRIPTION